test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -81,30 +122,13 @@ TEST_F(XdsOverrideHostTest, NoConfigReportsError) {
 }
 
 TEST_F(XdsOverrideHostTest, OverrideHost) {
-  const std::array<absl::string_view, 2> kAddresses = {"ipv4:127.0.0.1:441",
-                                                       "ipv4:127.0.0.1:442"};
-  EXPECT_EQ(ApplyUpdate(BuildUpdate(kAddresses,
-                                    MakeXdsOverrideHostConfig("round_robin")),
-                        policy_.get()),
-            absl::OkStatus());
-  ExpectConnectingUpdate();
-  // Ready up both subchannels
-  for (auto address : kAddresses) {
-    auto subchannel = FindSubchannel({address});
-    ASSERT_NE(subchannel, nullptr);
-    ASSERT_TRUE(subchannel->ConnectionRequested());
-    subchannel->SetConnectivityState(GRPC_CHANNEL_CONNECTING);
-    subchannel->SetConnectivityState(GRPC_CHANNEL_READY);
-  }
-  WaitForConnected();
-  ExpectState(GRPC_CHANNEL_READY);
-  auto picker = ExpectState(GRPC_CHANNEL_READY);
+  // Send address list to LB policy.
+  const std::array<absl::string_view, 3> kAddresses = {
+      "ipv4:127.0.0.1:441", "ipv4:127.0.0.1:442", "ipv4:127.0.0.1:443"};
+  auto maybe_picker = InitRoundRobin(kAddresses);
+  ASSERT_TRUE(maybe_picker.has_value());
+  auto picker = *maybe_picker;
```

Comment:
Instead of moving this into a different variable, you can just access it directly via the `absl::optional<>`.  Suggest writing it like this:

```
auto picker = InitRoundRobin(kAddresses);
ASSERT_TRUE(picker.has_value());
```

Then just change the code below to use `picker->get()` instead of `picker.get()`.

Same thing in all tests.